internal/http/services/owncloud/ocdav/error.go

```diff
@@ -58,13 +58,17 @@ type exception struct {
 
 // Marshal just calls the xml marshaller for a given exception.
 func Marshal(e exception) ([]byte, error) {
-	return xml.Marshal(&errorXML{
+	xmlstring, err := xml.Marshal(&errorXML{
 		Xmlnsd:    "DAV",
 		Xmlnss:    "http://sabredav.org/ns",
 		Exception: codesEnum[e.code],
 		Message:   e.message,
 		Header:    e.header,
 	})
+	if err == nil {
```

Comment:
@dragotin can you reverse the check? 😅
`if err != nil` and return `xmlstring` as the default? It's more go-idiomatic.

Reply:
Thanks for pushing me the right direction - changed!

Reply:
Thanks!